server/controller/login.js

```diff
@@ -0,0 +1,20 @@
+const bcrypt = require('bcryptjs');
+const { getuser } = require('../database/getuser');
+const gentoken = require('./gentoken');
+
+const login = (req, response) => {
+  const { email, password } = req.body;
+  getuser(email).then((res) => {
+    if (res.rows.length > 0) {
+      bcrypt.compare(password, res.rows[0].password).then((respa) => {
+        if (respa) {
+          gentoken(email).then((token) => {
+            response.cookie('email', token, { maxAge: 60 * 60 * 60 });
+            response.json('done');
```

Comment:
I would prefer that we send an object. I am not entirely sure that this is important, but I think the format of the response should be uniform across the app.

We should also set the status in the response, especially for errors and redirects...etc.